src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const newState = { ...state };
+  const arrayStates = [];
```

Comment:
Don't use names of data types in variable name
```suggestion
  const states = [];
```